intake/catalog/tests/test_catalog_save.py

```diff
@@ -10,6 +10,8 @@
 def test_catalog_description():
     """Make sure the description comes through the save."""
 
+    NAME = 'desc_test.yaml'
```

Comment:
OK, got it - so this should be a tmpdir/tmpfile location

Reply:
Changed this to use tmpdir and removed the explicit delete.